README.md

```diff
@@ -472,6 +472,15 @@ In detail, the tags work as follows:
                       a positive value will send exactly x packets (possibly
                       with acks if ack is true), while a negative number will
                       send a random number of packets between 1 and abs(-x).
+                    - ttl: set time to live value for packet. By default,
+                      sniffles will generate random TTL value.
+                    - ttl_expiry: simulate the ttl expiry attack by breaking
+                      packets into multiple packet with one malicious packet 
```

Comment:
It is a good habit not to leave space characters at the end of a line. It matters little in this case since it is a text file, but it could be annoying to see a lot of diffs in a pull request due to whitespace changes. Please delete the trailing whitespace in this line and the next.